<launch>   
    <!-- launch video stream -->
    <include file="$(find video_stream_opencv)/launch/camera.launch" >
        <arg name="camera_name" value="camera" />
        <arg name="video_stream_provider" value="0" />
        <arg name="set_camera_fps" value="30"/>
        <arg name="buffer_queue_size" value="1" />
        <arg name="fps" value="30" />
        <arg name="frame_id" value="webcam_optical_frame" />
        <arg name="camera_info_url" value="" />
        <arg name="flip_horizontal" value="false" />
        <arg name="flip_vertical" value="false" />
        <arg name="visualize" value="false" />
    </include>

    <node name="image_proc" pkg="image_proc" type="image_proc">
        <remap from="image_raw" to="/camera/image_raw"/>
        <remap from="camera_info" to="/camera/camera_info"/>
        <remap from="image_rect_color" to="/camera/image_rect_color"/>
    </node>

    <!-- <node name="image_view" pkg="image_view" type="image_view" args=" image:=/camera/image_rect_color"/> -->
    <node name="face_detector" pkg="demo_3_march" type="face_detector.py">
        <param name="short_check_limit" value="5"/>
        <param name="long_check_limit" value="20"/>
    </node>

    <node name="main" pkg="demo_3_march" type="demo.py"/>
</launch>